packages/grplot.yml

```
@@ -0,0 +1,5 @@
+repo: 'ghiffaryr/grplot'
+section: 'domain specific libraries'
+description: 'lazy statistical data visualization.'
```

```suggestion
description: 'Lazy statistical data visualization.'
```
Everything else is caps, so would be more standard- also is there anything more descriptive than lazy?

---

Okay, thanks, and nope since it is intended for lazy people to draw complete and attractive statistical graphs